modin/experimental/engines/omnisci_on_ray/frame/partition_manager.py

```diff
@@ -47,17 +48,95 @@ def _compute_num_partitions(cls):
         return 1
 
     @classmethod
-    def from_arrow(cls, at, return_dims=False):
+    def from_pandas(cls, df, return_dims=False, unsupported_cols=None):
```

Comment:
What is the purpose of the `unsupported_cols` parameter? I don't see a scenario where it is really used by a caller.

Reply:
that was added to be consistent with `from_arrow`, to where I've added that parameter to be able to skip check for unsupported cols, in `from_pandas` we don't use it, so I've removed that parameter